switchbot/const.py

```diff
@@ -71,3 +72,35 @@ class LockStatus(Enum):
     LOCKING_STOP = 4  # LOCKING_BLOCKED
     UNLOCKING_STOP = 5  # UNLOCKING_BLOCKED
     NOT_FULLY_LOCKED = 6  # LATCH_LOCKED - Only EU lock type
```

Comment:
not for this PR, but its a bit strange that we have lock data in `const.py` and not in the lock platforms

Reply:
Please take a look at #303 I did extract that part in that PR in order to do the same here